src/D2L.Bmx/UpdateChecker.cs

```diff
@@ -0,0 +1,119 @@
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace D2L.Bmx;
+internal class UpdateChecker {
```

Comment:
```suggestion
internal static class UpdateChecker {
```

This class is a collection of helper statics, not to be instantiated